From 3ae48d4df9b985ae0c93d3e493bf8362ef9c1094 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 15 Nov 2024 20:40:17 +0800
Subject: [PATCH 2/2] limits.conf: add default settings for AOSC OS

- Set max realtime priority to 0 for all users.
- Set max nice value to 0 for all users.
- Set max realtime priotity to 65 for users of the audio group.
- Set max nice value to -10 for users of the audio group.
- Set max locked-in-memory address space to 40,000KiB for users of the
  audio group.
---
 modules/pam_limits/limits.conf | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/modules/pam_limits/limits.conf b/modules/pam_limits/limits.conf
index e8a746cc..44972ee4 100644
--- a/modules/pam_limits/limits.conf
+++ b/modules/pam_limits/limits.conf
@@ -58,4 +58,8 @@
 #ftp             hard    nproc           0
 #@student        -       maxlogins       4
 
-# End of file
+*		-	rtprio		0
+*		-	nice		0
+@audio		-	rtprio		65
+@audio		-	nice		-10
+@audio		-	memlock		40000
-- 
2.47.0

